R/pkg/R/mllib.R

```diff
@@ -917,14 +922,14 @@ setMethod("spark.lda", signature(data = "SparkDataFrame"),
 # Returns a summary of the AFT survival regression model produced by spark.survreg,
 # similarly to R's summary().
 
-#' @param object A fitted AFT survival regression model
+#' @param object a fitted AFT survival regression model.
 #' @return \code{summary} returns a list containing the model's coefficients,
 #' intercept and log(scale)
 #' @rdname spark.survreg
 #' @export
 #' @note summary(AFTSurvivalRegressionModel) since 2.0.0
 setMethod("summary", signature(object = "AFTSurvivalRegressionModel"),
-          function(object, ...) {
+          function(object) {
```

Comment:
We have `...` for `summary`? That is used to match the `base::summary` signature. I am not completely sure about the exact reason, but I read from the [doc for Methods](https://stat.ethz.ch/R-manual/R-devel/library/methods/html/Methods.html) saying 

"By default, the signature of the generic consists of all the formal arguments except ..., in the order they appear in the function definition." 

Does that perhaps explain that behavior?